### PR TITLE
Updated Consistency03

### PR DESCRIPTION
**Pull request by matsduf**
* Updated the headline.
* Updated to new format.
* Added explicit messages.
* Removed copyright statement.
* Removed description/headline from identifier line.

Changes compared to current implementation (develop branch pre v2018.1): **none**

### CODE REVIEW

File: docs/specifications/tests/Consistency-TP/consistency03.md [hunk 1]
```diff
@@ -1,61 +1,87 @@
...
- 7. Verify that all SOA records have the same EXPIRE value.
- 8. Verify that all SOA records have the same MINIMUM value.
+ 1. Obtain the list of name server IPs for the *child zone* from [Method4] 
+    and [Method5].
 
+ 2. Create an SOA query for *child zone* apex and send it to all name 
+    server IPs.
 
-### Outcome(s)
+ 3. Retrieve the SOA RR from the responses from all name server IPs.
```
Comment (mattias-p) [inline docs/specifications/tests/Consistency-TP/consistency03.md]: Have you considered rearranging the steps to collect responses first, check for missing responses second, and inspecting the received responses third? To me that's a more natural order.

File: docs/specifications/tests/Consistency-TP/consistency03.md [hunk 2]
```diff
@@ -1,61 +1,87 @@
...
-### Special procedural requirements	
+ 5. If a name server responds but not include a SOA record in the 
+    response, emit *[NO_RESPONSE_SOA_QUERY]*.
+
+ 6. If at least one SOA record has been retrieved, emit 
+    *[ONE_SOA_TIME_PARAMETER_SET]* if all SOA records have 
+    1. the same REFRESH value, 
+    2. the same RETRY value, 
+    3. the same EXPIRE value, and 
+    4. the same MINIMUM value.
```
Comment (mattias-p) [inline docs/specifications/tests/Consistency-TP/consistency03.md]: Have you considered the following wording?

> "Emit *[ONE_SOA_TIME_PARAMETER_SET]* if at least one SOA record has been retrieved and all SOA records have:"

Having fewer "if" clauses makes it easier to understand.

**Comment by matsduf** [comment]
@sandoche2k, can you review?

**Comment by matsduf** [comment]
Updated after review by @JohanIIS.

@mattias-p, can you approve again? @JohanIIS and @sandoche2k, please review.

File: docs/specifications/tests/Consistency-TP/consistency03.md [hunk 3]
```diff
@@ -6,25 +6,25 @@
...
 across all authoritative name servers.
 
-The inconsistency in these different fields for the designated zone
-might result in operational inconsistencies.
+An inconsistency in these fields might result in operational inconsistencies 
+for the designated zone.
 
 There are other test cases that provide consistency tests for other
-SOA fields.
+SOA fields. **(Suggest either mentioning which test cases this refers to, or removing this statement.)**
```
Comment (mattias-p) [inline docs/specifications/tests/Consistency-TP/consistency03.md]: I suggest removing this statement.

Comment (matsduf) [reply docs/specifications/tests/Consistency-TP/consistency03.md]: That statement was a comment from @JohanIIS to me and has already been removed through my additional updates some 10 minutes after I merged @JohanIIS 's commit into my branch.

**Pull request merged**